src/components/imgix/imgixItems/ImgixItems.jsx

```diff
@@ -126,13 +46,12 @@ export default function ImgixItems() {
         Items
       </h1>
       <div className='items'>
-        {Object.keys(params).map((i) => {
+        {Object.keys(imageFilters).map((i) => {
```

Comment:
You can iterate directly on the elements, as map receives as parameters the element and the index.

Reply:
I need the keys in this case, so Object.keys (currently being used) is the right approach